src/Simplex/Chat/Store.hs

```diff
@@ -789,47 +791,47 @@ deleteUserAddress db User {userId} = do
     [":user_id" := userId]
   DB.execute db "DELETE FROM user_contact_links WHERE user_id = ? AND local_display_name = '' AND group_id IS NULL" (Only userId)
 
-getUserAddress :: DB.Connection -> UserId -> ExceptT StoreError IO (ConnReqContact, Bool, Maybe MsgContent)
+getUserAddress :: DB.Connection -> UserId -> ExceptT StoreError IO (ConnReqContact, Bool, Bool, Maybe MsgContent)
 getUserAddress db userId =
   ExceptT . firstRow id SEUserContactLinkNotFound $
     DB.query
       db
       [sql|
-        SELECT conn_req_contact, auto_accept, auto_reply_msg_content
+        SELECT conn_req_contact, auto_accept, incognito, auto_reply_msg_content
         FROM user_contact_links
         WHERE user_id = ? AND local_display_name = '' AND group_id IS NULL
       |]
       (Only userId)
 
-getUserContactLinkById :: DB.Connection -> UserId -> Int64 -> IO (Maybe (ConnReqContact, Bool, Maybe MsgContent, Maybe GroupId))
+getUserContactLinkById :: DB.Connection -> UserId -> Int64 -> IO (Maybe (ConnReqContact, Bool, Bool, Maybe MsgContent, Maybe GroupId))
 getUserContactLinkById db userId userContactLinkId =
   maybeFirstRow id $
     DB.query
       db
       [sql|
-        SELECT conn_req_contact, auto_accept, auto_reply_msg_content, group_id
+        SELECT conn_req_contact, auto_accept, incognito, auto_reply_msg_content, group_id
         FROM user_contact_links
         WHERE user_id = ?
           AND user_contact_link_id = ?
       |]
       (userId, userContactLinkId)
 
-updateUserAddressAutoAccept :: DB.Connection -> UserId -> Bool -> Maybe MsgContent -> ExceptT StoreError IO (ConnReqContact, Bool, Maybe MsgContent)
-updateUserAddressAutoAccept db userId autoAccept msgContent = do
-  (cReqUri, _, _) <- getUserAddress db userId
+updateUserAddressAutoAccept :: DB.Connection -> UserId -> Bool -> Bool -> Maybe MsgContent -> ExceptT StoreError IO (ConnReqContact, Bool, Bool, Maybe MsgContent)
```

Comment:
we've been kicking this can down the road long enough - it's time to make a record that would be also returned as a single parameter of CRUserContactLink and the next one...

Two options here:

1. extend UserAddress in types:
2. create a new one UserContactLink

```
data UserContactLink = UserContactLink
  { userContactLinkId :: Int64,
    connReqContact :: ConnReqContact,
    autoAccept :: Bool,
    autoAcceptIncognito :: Bool,
    autoReply :: Maybe MsgContent
  }
```